packages/components/managed-data-table/src/data-table-settings/column-manager/column-manager.spec.js

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import { screen, render } from '../../../../../../test/test-utils';
+import PrimaryButton from '../../../../buttons/primary-button';
+import SecondaryButton from '../../../../buttons/secondary-button';
+import { ColumnManager, handleDragEnd } from './column-manager';
+import { HIDDEN_COLUMNS_PANEL, SELECTED_COLUMNS_PANEL } from './constants';
+
+const createTestProps = (props) => ({
+  availableColumns: [],
+  selectedColumns: [],
+  onUpdateColumns: jest.fn(),
+  ...props,
+});
+
+describe('ColumnManager', () => {
+  it('should render hidden columns', () => {
+    const props = createTestProps({
+      availableColumns: [
+        {
+          key: 'column1',
+          label: 'Column 1',
+        },
+      ],
+    });
+
+    render(<ColumnManager {...props} />);
+
+    expect(screen.getByText(/Column 1/i)).toBeInTheDocument();
+  });
+
+  it('should render visible columns', () => {
+    const props = createTestProps({
+      selectedColumns: [
+        {
+          key: 'column1',
+          label: 'Column 1',
+        },
+      ],
+      availableColumns: [
+        {
+          key: 'column1',
+          label: 'Column 1',
+        },
+      ],
+    });
+
+    render(<ColumnManager {...props} />);
+
+    expect(screen.getByText(/Column 1/i)).toBeInTheDocument();
+  });
+
+  describe('when `disableHiddenColumnSearch` is true', () => {
+    it('should render columns search input', () => {
+      const props = createTestProps({
+        selectedColumns: [
+          {
+            key: 'column1',
+            label: 'Column 1',
+          },
+        ],
+        availableColumns: [
+          {
+            key: 'column1',
+            label: 'Column 1',
+          },
+        ],
+        disableHiddenColumnSearch: false,
```

Comment:
The test description says `true` but here you pass `false`. Which one is it?

Reply:
Forgot to change the description after we change it from enabled to disabled